### DISCUSSION TITLE
Warner TV Italy

### DISCUSSION THREAD

**Discussion by fparri** [iptv-org/community#849] [Q&A]
### Channel ID

WarnerTV.it

### Notes

Anyone can add this channel? It can be streamed online here: https://warnertv.it/live-streaming-warner-tv

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Quite a few steps to do : 

https://eu1-prod.disco-api.com/token?realm=dplay -> https://eu2-prod.disco-api.com/playback/v3/videoPlaybackInfo (POST {"deviceInfo":{"adBlocker":false,"drmSupported":true,"hdrCapabilities":["SDR"],"hwDecodingCapabilities":[],"soundCapabilities":["STEREO"]},"wisteriaProperties":{},"videoId":"178906"}) -> data.streaming[0].url

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19794
